pkg/lib/cockpit-po-plugin.js

```diff
@@ -111,8 +109,19 @@ function init(options) {
 function run(webpack_module, webpack_compilation) {
     const promises = [];
     config.subdirs.map(subdir =>
-        promises.push(...get_po_files().map(po_file =>
-            buildFile(po_file, subdir, webpack_module, webpack_compilation))));
+        promises.push(...get_po_files().map(f => {
```

Comment:
Using a lambda in a map which does side effects via its closure is really double plus not good.  I'd love to see this turned one more layer into a for loop....